scripts/dependencies/wsl.ts

```diff
@@ -28,6 +28,38 @@ function extract(resourcesPath: string, file: string, expectedFile: string): voi
   fs.rmSync(file, { maxRetries: 10 });
 }
 
+export class HostSwitch implements Dependency, GithubDependency {
+  name = 'hostSwitch';
+  githubOwner = 'rancher-sandbox';
+  githubRepo = 'rancher-desktop-networking';
+
+  async download(context: DownloadContext): Promise<void> {
+    const baseURL = `https://github.com/${ this.githubOwner }/${ this.githubRepo }/releases/download`;
+    const zipName = `rancher-desktop-networking-v${ context.versions.rdNetworking }.tar.gz`;
```

Comment:
Just realized that `zipName` doesn't point to a zip file… :D

Reply:
Good catch, of course, I changed how we create the release archive now 😄